assets/content/documentation/H2d/5. Displaying-Text.md

```diff
@@ -0,0 +1,65 @@
+# Displaying text
+
+`h2d.Text` can be used to draw text using bitmap fonts. It enables you to set textColor, maxWidth, textAlign, letterSpacing and lineSpacing.
+It's also possible to get the size of the text with textWidth/textHeight.
+
+### Use text in Heaps
+
+```haxe
+var tf = new h2d.Text(hxd.Res.fonts.myFontName.toFont());
+tf.text = "Hello World\nHeaps is great!";
+tf.align = Center;
+
+// add to any parent; in this case we append to root
+s2d.addChild(tf);
+```
+
+For each font family you need a different font file, but you can tint them with code, using `textColor`.
+
+## Font creation
+
+### Convert font to bitmap font using BMfont tool
+
```

Comment:
At Shiro we are using FontBuilder, maybe good to document it as well? 
https://github.com/HeapsIO/fontbuilder/releases (win + osx, no linux)